src/utils/bugzilla/getBugsData.js

```diff
@@ -86,8 +17,37 @@ const getBugsData = async (queries = [], startDate) => {
       };
     })
   );
-
-  return chartJsFormatter(data, startDate);
+  const data = selectFrom(timeDomain.partitions)
+    .map(p => ({
+      date: p.min,
+      ...selectFrom(bugSeries)
+        .map(({ bugs, label }) => [
+          selectFrom(bugs)
+            .filter(({ cf_last_resolved, creation_time }) => {
+              const end = Date.newInstance(cf_last_resolved);
+              const start = Date.newInstance(creation_time);
+
+              return !(end < p.min || p.max < start);
+            })
+            .count(),
```

Comment:
if the bug is alive on the given day, then include it in the count of bugs for the day